src/tech/v3/dataset/print.clj

```diff
@@ -216,3 +216,31 @@ tech.ml.dataset.github-test> (def ds (with-meta ds
              (dataset-data->str ds options))))
   ([ds]
    (dataset->str ds {})))
+
+
+(defn print-range
+  "Convenience function to set the number of rows to print."
+  [dataset index-range]
+  (-> dataset
+      (vary-meta assoc :print-index-range index-range)))
+
+
+(defn print-policy
```

Comment:
Maybe mention the available policies in the comment or refer to the documentation for print.